### ISSUE TITLE
Add hot start to app homescreen test for Fennec 68

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/perf-frontend-issues#24]
Blocked by?:
- cold/warm/hot definitions: https://github.com/mozilla-mobile/fenix/issues/5910
- Fennec 68 cold start: #23 
- Fenix hot start: https://github.com/mozilla-mobile/fenix/issues/6508

### Why/User Benefit/User Problem
We want to compare the performance of Fenix against Fennec 68 as a baseline. A key metric is hot startup to the homescreen: let's make an equivalent startup test for Fennec.

### Impact
Without this, we cannot establish baseline performance or measure our proposed release criteria.

### Acceptance Criteria (how do I know when I’m done?)
- Fennec 68 has hot startup to app homescreen test equivalent to Fenix
- Link to dashboard is added to perf sync document
- ? If needed, file follow-up to add this information into our dashboards

**Comment by mcomella** [comment]
(copy-paste) Not a priority: closing.
